packages/tizen_rpc_port/tizen/src/rpc_port_proxy.h

```diff
@@ -0,0 +1,53 @@
+// Copyright 2022 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef RPC_PORT_PROXY_H
+#define RPC_PORT_PROXY_H
+
+#include <flutter/event_channel.h>
+#include <flutter/standard_method_codec.h>
+#include <rpc-port-parcel.h>
+#include <rpc-port.h>
+
+#include <map>
+#include <memory>
+#include <set>
+#include <string>
+#include <vector>
+
+#include "common.h"
+
+namespace tizen {
+
+typedef std::unique_ptr<flutter::EventSink<flutter::EncodableValue>> EventSink;
+
+class RpcPortProxyManager {
+ public:
+  static RpcPortProxyManager& GetInst();
```

Comment:
Please do not use meyers singleton pattern in the libary code.
It makes crash issues while terminating the process.

Reply:
I overlooked this case.
Thank you for your review.